java/awssdk-autoconfigure/src/main/resources/software/amazon/awssdk/global/handlers/execution.interceptors

```diff
@@ -0,0 +1 @@
+io.opentelemetry.instrumentation.awssdk.v2_2.autoconfigure.AutoconfiguredTracingExecutionInterceptor
```

Comment:
How does it work? looks like SPI

Reply:
Yeah it's an AWS SDK specific format for SPI